frontend/src/app/modules/user-profile/user-details/user-details/user-details.component.ts

```diff
@@ -102,15 +125,23 @@ export class UserDetailsComponent extends BaseComponent implements OnInit {
         return this.detailsForm.get('email') as FormControl;
     }
 
-    get instagram(): FormControl {
-        return this.detailsForm.get('instagram') as FormControl;
+    selectInterest($event: Event, i: number) {
+        const ev = $event.target as HTMLInputElement;
```

Comment:
event should be a Tag interface, not HTMLInputElement

Reply:
fixed